connector/connect/server/src/main/scala/org/apache/spark/sql/connect/service/SparkConnectService.scala

```diff
@@ -315,6 +315,12 @@ object SparkConnectService extends Logging {
       previoslyObservedSessionId)
   }
 
+  // For testing
+  private[spark] def getOrCreateIsolatedSession(
```

Comment:
Where are you using this?

Reply:
I use this for testing at https://github.com/apache/spark/pull/46570#discussion_r1599971309. Basically I launches a Spark Connect server via Py4J, and make a Spark Connect client to make requests to them. After that, I get this from the Spark Connect server via Py4J.